pkg/controller/apikey/index.go

```diff
@@ -48,9 +48,21 @@ func (c *Controller) HandleIndex() http.Handler {
 		}
 
 		m := controller.TemplateMapFromContext(ctx)
+
+		creationCounts := make(map[uint]int64)
+		for _, app := range realm.AuthorizedApps {
+			count, err := c.db.CountVerificationCodesByAuthorizedApp(app.ID)
+			if err != nil {
+				flash.Error("Error loading app code creation counts: %v", err)
```

Comment:
Do we mean to continue after this error or is the fall through to normal case intentional?

Reply:
removed these. I had them while dev-ing since I was getting intermittent errors